### PR TITLE
chore: Update Prow/Lighthouse comparison

### PR DESCRIPTION
**Pull request by abayer**
Signed-off-by: Andrew Bayer <andrew.bayer@gmail.com>

### CODE REVIEW

**Comment by abayer** [comment]
Blog post still to come later today or tomorrow morning, but I thought I'd get this up ASAP after @jglick made me think to do so. =)

File: content/en/docs/reference/components/lighthouse.md [hunk 1]
```diff
@@ -76,28 +74,24 @@ webhook: lighthouse
...
-* lighthouse focuses purely on Tekton pipelines so it does not require a `ProwJob` CRD; instead a push webhook to a release or pull request branch can trigger zero to many `PipelineRun` CRDs instead
+Lighthouse is based on a fork of Prow's source code, including most of the built-in [plugins from prow](https://github.com/jenkins-x/lighthouse/tree/master/pkg/plugins)
 
+The most noteworthy differences are:
 
+* In order to support multiple SCM providers, Lighthouse uses [jenkins-x/go-scm](https://github.com/jenkins-x/go-scm) as an abstraction layer, allowing additional provider support to be implemented without needing significant changes within Lighthouse itself.
+* Lighthouse has less moving parts and potential configuration than Prow, resulting in only four pods being run for Lighthouse: two webhook receiver replicas, a Keeper pod, which handles deciding when to merge and actually merging pull requests, and a Foghorn pod, which handles reporting commit statuses back to the provider.
```
Comment (jglick) [inline content/en/docs/reference/components/lighthouse.md:82]: “Keeper”? No nautical metaphor, like “Tugboat” or something? :-)

Comment (abayer) [reply content/en/docs/reference/components/lighthouse.md:82]: "Lighthouse Keeper". =)

**Pull request merged**